calico_containers/calico_ctl/node.py

```diff
@@ -13,24 +13,28 @@
 # limitations under the License.
 """
 Usage:
-  calicoctl node [--ip=<IP>] [--ip6=<IP6>] [--node-image=<DOCKER_IMAGE_NAME>] [--as=<AS_NUM>] [--log-dir=<LOG_DIR>] [--detach=<DETACH>] [--kubernetes] [--rkt] [--libnetwork]
+  calicoctl node [--ip=<IP>] [--ip6=<IP6>] [--node-image=<DOCKER_IMAGE_NAME>]
+    [--as=<AS_NUM>] [--log-dir=<LOG_DIR>] [--detach=<DETACH>]
+    [--kubernetes] [--rkt]
+    [(--libnetwork [--libnetwork-image=<LIBNETWORK_IMAGE_NAME>])]
```

Comment:
it forces the --libnetwork-image to only be valid when --libnetwork is also specified.

without it you can specify the image without the --libnetwork flag which i think is misleading.